lib/integrations/slack/incoming_message_builder.rb

```diff
@@ -77,7 +77,7 @@ def sender
   end
 
   def private_note?
-    params[:event][:text].strip.starts_with?('note:', 'private:')
+    params[:event][:text].strip.starts_with?('note:', 'private:', 'Note:', 'Private:')
```

Comment:
@Ysajid Thanks for the contribution. I would suggest to downcase instead of Checking for `Note:` or `Private:`. Let me know what you think.


```suggestion
    params[:event][:text].strip.downcase.starts_with?('note:', 'private:')
```

Reply:
Yes, that'd be better.